etc/kayobe/ansible/vault-deploy-overcloud.yml

```diff
@@ -60,10 +60,15 @@
     - import_role:
         name: stackhpc.hashicorp.vault
       vars:
+        hashicorp_registry_url: "{{ overcloud_hashicorp_registry_url }}"
```

Comment:
Would it make sense to do the same for the seed vault to keep them in sync?

Reply:
I have updated the pull request to support local images for the seed. Can you please review again?